frontend/src/App.js

```diff
@@ -106,7 +112,16 @@ class App extends React.Component {
         <Login parent={this} gameState={this.state.gameState} />
         {this.state.showGame === true && (
           <div>
-            <div className="topBar"></div>
+            <div className="topBar">
+              <div className="endGameButton" onClick={this.handleReset}>
```

Comment:
If you wanted, you could inline something like the following here () => tools.resetPlayers(this) to avoid declaring the extra method. Think inlining is reasonable in cases where the method is very short.

Don't have a strong opinion.